packages/components/color-picker/tests/color-picker.test.tsx

```diff
@@ -47,4 +47,20 @@ describe("<ColorPicker />", () => {
 
     resetEyeDropperMock()
   })
+
+  test("ColorPicker renders as disabled", () => {
+    render(<ColorPicker data-testid="colorPicker" disabled />)
+
+    const colorPicker = screen.getByTestId("colorPicker")
+
+    expect(colorPicker).toHaveAttribute("disabled")
```

Comment:
@108yen 
thanks for your reivew!

Do you mean to use "textbox" instead of "testID" and "toBeDisabled" instead of "toHaveAttribute"?
or 
Do you mean to use "fireEvent" or "userEvent" instead?

Reply:
I did not understand the meaning of this question.
> Do you mean to use "fireEvent" or "userEvent" instead?
Please use `getByRole` as much as possible.

Reply:
fixed it